app/views/collections/show.html.erb

```diff
@@ -57,55 +57,53 @@
       </div>
     <% else %>
       <section class="action">
-        <div class="cta">
```

Comment:
these changes seem much bigger than they actually are, it appears that way because I removed the `<div class="cta">` and had to outdent everything within it